superset/assets/stylesheets/superset.less

```diff
@@ -503,6 +503,24 @@ tr.reactable-column-header th.reactable-header-sortable {
   text-align: center;
 }
 
+.overlay-chart {
+  font-weight: bold;
```

Comment:
font-styles should be specific to label, not the whole chart-container

Reply:
Done